fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResource.java

```diff
@@ -701,8 +699,8 @@ public String deleteLoanApplication(@PathParam("loanId") final Long loanId) {
     @Path("{loanId}")
```

Comment:
mention about all cases in the notes section and 
show response to one query!
and mention in notes that available response is for given query
for all query refer to current docs with it's link